.github/workflows/test-forks.yml

```diff
@@ -25,13 +25,7 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
 
-      - uses: anybody/setup-anybody@main
-        name: Install AnyBody
-        with:
-          anybody-version: "7.5.0"
-          anybody-version-suffix: "Beta"
-
-      - uses: prefix-dev/setup-pixi@v0.4.3
+      - uses: prefix-dev/setup-pixi@v0.5.0
         id: pixi
         continue-on-error: true
         with:
```

Comment:
you need `environments: test` here. Otherwise it will only cache the `default` environment.